services/QuillLMS/engines/comprehension/lib/comprehension/comprehension/change_log.rb

```diff
@@ -0,0 +1,17 @@
+module Comprehension
+  module ChangeLog
```

Comment:
I suspect you can move `has_many :change_logs` and the `after_save` hook into this module, via an `included?` callback. This greatly reduces the touchpoints for AR models that include this module.

Docs: https://ruby-doc.org/core-2.6.3/Module.html#method-i-included

Reply:
(we zoomed to discuss) Cissy is discuss with Lindsey, the specification writer, on whether we can change the spec to make the AR callbacks more standardized across classes, thus reducing the complexity of each integration (an integration is an AR module that includes the ChangeLog module)

Reply:
I've added in the ChangeLog module includes feature as discussed!